Model/Cc.php

```diff
@@ -690,14 +690,17 @@ protected function callCurl($url, $params, $method = "POST")
      */
     public function processErrors($response)
     {
-
         if (isset($response['Error']) && count($response['Error'])) {
-            $errorMessage = $response['Error']['Code'];
-            if (isset($response['Error']['Message']) && $response['Error']['Message'] != '') {
-                $errorMessage = ':'.$response['Error']['Message'];
+            $error = $response['Error']['Code'];
+            if (isset($error) && $error != '') {
+                if ($error == "Blacklisted" || $error == "FraudSuspected") {
+                    $errorMessage = ': PaymentAuthorizationFailed';
+                } else {
+                    $errorMessage = ':'.$error;
```

Comment:
Are the `':'` necessary? I think when we concat the string on line `703` we'll end up with double `:`

Reply:
This was what was here previously with the other errors that I had seen.

Reply:
Ohh I see we might want to remove the colon in `Something went wrong while generating the payment request:` then or not include the `:` in the values set for `$errorMessage`

Reply:
updated!